airflow_kubernetes_job_operator/kubernetes_job_operator.py

```diff
@@ -224,7 +263,9 @@ def execute(self, context):
         Raises:
             AirflowException: Error in execution.
         """
-
+        assert self._job_runner is not None, KubernetesJobOperatorException(
+            "Execute called without pre_execute. Job runner is object was not created."
```

Comment:
Remove "is" -> `Job runner object was not created`.

Reply:
- [ ] Fix comment